target/linux/ramips/image/mt76x8.mk

```diff
@@ -167,6 +167,26 @@ define Device/mercury_mac1200r-v2
 endef
 TARGET_DEVICES += mercury_mac1200r-v2
 
+define Device/netgear_r6080
+  BLOCKSIZE := 64k
+  IMAGE_SIZE := 7872k
```

Comment:
I calculated 7552. You would have 7b0000 ...

Reply:
You're right, bad math by me. Fixed.